Apps/Admin/Client/Store/Dashboard/DashboardActions.cs

```diff
@@ -29,13 +29,13 @@ public static class DashboardActions
     /// <summary>
     /// The action representing the initiation of registered users load action.
     /// </summary>
-    public class LoadRegisteredUsersAction
+    public class RegisteredUsersAction
```

Comment:
Nit: I prefer that the names include Load, since that is what these are all doing.